NineAnimator/Controllers/RootViewController.swift

```diff
@@ -247,46 +247,54 @@ extension RootViewController {
             preferredStyle: .alert
         )
         
-        let errorAlert = UIAlertController(
-            title: "Error",
-            message: "Cannot import configurations",
-            preferredStyle: .alert
-        )
-        
-        errorAlert.addAction(UIAlertAction(
-            title: "Done",
-            style: .cancel,
-            handler: nil
-        ))
-        
-        func showErrorAlert() { presentOnTop(errorAlert) }
+        func showErrorAlert(error: Error) {
+            Log.error(error)
+            let errorAlert = UIAlertController(
+                title: "Error",
+                message: "Cannot import configurations: \(error)",
+                preferredStyle: .alert
+            )
+            
+            errorAlert.addAction(UIAlertAction(
+                title: "Done",
+                style: .cancel,
+                handler: nil
+            ))
```

Comment:
Use `UIAlertController(error:, customTitle:, allowRetry:, retryActionName:, source:, completionHandler:)` if possible (see `UIAlertController+Error.swift` under the Utilities group).